etl/steps/data/garden/oecd/2025-03-04/health_expenditure_1993.meta.yml

```diff
@@ -0,0 +1,33 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Healthcare Spending
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
```

Comment:
Perhaps should be 0 as I don't expect we'll update this

Reply:
Yes, thanks! The wizard only allows to put a date in there